tests/phpunit/ErrorResponseHandlingTest.php

```diff
@@ -111,15 +125,43 @@ public function dataProviderBatchRequest(): Generator
                 ),
             ],
             'expectedMessage' => 'OneDrive API error: The service is unavailable.',
+            'checkIfRetires' => false,
         ];
 
         yield 'Request took to long 504 error' => [
             'responses' => [
                 $this->getLoadSheetListSuccessResponse(),
                 $this->getBatchErrorResponse(504, 'MaxRequestDurationExceeded', "We'" .
```

Comment:
Hoď jsem asi ještě pls nějaký `array_fill` jako je to v těch funkcích dole, ať tu není x stejných řádků